Misleading/confusing error message when not in git repository

### Description

Running in a non-git directory results in a panic error with a message that is a little bit confusing because of some pathing that does not belong to my machine. 

```sh
╰──➤ ./broom
panic: repository does not exist

goroutine 1 [running]:
main.InitializeMenu()
	/home/anthony/projects/broom/broom.go:28 +0x11cc
main.main()
	/home/anthony/projects/broom/main.go:4 +0x1c
```

### Proposed Solution
I would expect the stderr for executing in a non-git repository to let me know the current pwd path is not a git repository and have a clean exit message.

--- a/broom.go
+++ b/broom.go
@@ -2,6 +2,7 @@ package main
 
 import (
 	"fmt"
+	"log"
 	"os"
 
 	"github.com/gdamore/tcell/v2"
@@ -12,6 +13,8 @@ func InitializeMenu() {
 
 	var branchesToDelete []string
 	var path string
+        
+        pwd, _ := os.Getwd()
 
 	modalIsOpen := false
 
@@ -25,12 +28,14 @@ func InitializeMenu() {
 
 	repo, err := NewGitRepositoryFromString(path)
 	if err != nil {
-		panic(err)
+          error := fmt.Errorf("error for path %s: %s", pwd, err)
+          log.Fatal(error)
 	}
 
 	refs, err := NewReferences(repo)
 	if err != nil {
-		panic(err)
+          error := fmt.Errorf("error: %s", err)
+          log.Fatal(error)
 	}
 
 	helpBox := tview.NewTextView()

--- a/flake.nix
+++ b/flake.nix
@@ -36,6 +36,12 @@
         };
       in
       {
+        devShells = {
+          default = pkgs.mkShell {
+            packages = [ pkgs.go ];
+            inputsFrom = [ self.packages ];
+          };
+        };
         packages = {
           inherit broom;
         };

